rostwitter/python/rostwitter/twitter.py

```diff
@@ -73,19 +73,40 @@ def _check_and_split_word(self, text):
                 text = ''
         return text
 
+    def _post_update_with_reply(self, texts, in_reply_to_status_id=None):
+        for text in texts:
+            url = 'https://api.twitter.com/1.1/statuses/update.json'
+            data = {'status': StringIO(text)}
+            if in_reply_to_status_id is not None:
+                data['in_reply_to_status_id'] = in_reply_to_status_id
+            json = self._request_url(url, 'POST', data=data)
+            data = simplejson.loads(json.content)
+            in_reply_to_status_id = data['id']
+        return data
+
     def post_update(self, status):
-        status = self._check_and_split_word(status)
```

Comment:
can we remove `_check_and_split_word` function too?

Reply:
Thanks for your feedback.
I removed the function to reduce diff of this PR.
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/373/commits/ad282826a9f803ad183974036dafe0492ee3dda9